activities/titilayo_tl.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+import os
+
+import grass.script as gs
+
+def run_overlandflow(scanned_elev, env, **kwargs):
+    gs.run_command("v.to.rast", input="streams@PERMANENT", output="rural_streams", use="val", val=1, flags="d")
+    gs.run_command("r.lake.series", elevation="elev_lid792_1m", seed="rural_streams", start_water_level=104.0, end_water_level=114.0, water_level_step = 0.2, output="flooding")
```

Comment:
You should have `scanned_elev` here instead of `elev_lid792_1m` so it will work as a generic Tangible Landscape activity.

```suggestion
    gs.run_command("r.lake.series", elevation=scanned_elev,  seed="streams", start_water_level=0, end_water_level=5, water_level_step = 0.5, output="flooding")
```